build/package.json

```diff
@@ -1,8 +1,11 @@
 {
```

Comment:
What do you think about making this a part of the `npm run build` task? We could copy the root `package.json` into the build directory, that way we would only have one `package.json` to maintain.